Onboarder (#1021)
akeneo/pim-docs: onboarder/synchronization/index.rst

Synchronization
===============

Push catalog structure to the retailer Onboarder
------------------------------------------------

To work, the retailer Onboarder needs some data from the PIM and you will have to push them once your installation will be done.

Data synchronized are: attribute groups, attributes, attribute option, categories, families and users.

.. code-block:: bash

    $ php bin/console akeneo:synchronization:push-catalog-to-onboarder --env=prod

.. note::

>     You will have to run this command only one time.

Reviewer: one time -> once

>     You will have to run this command only one time.

Reviewer 2: ```suggestion
    You will have to run this command only once.
```

>     Once the extension correctly installed, all data mentioned above will be synchronized automatically when they will be saved.

Reviewer 2: ```suggestion
    Once the extension is correctly installed, all data mentioned above will be synchronized automatically when they will be saved.
```

Launch the message worker
-------------------------

| The synchronization of the catalog structure and the catalog data is handled in a asynchronous way using the Google Cloud PubSub service.
| The messages that are part of the synchronization process are queued.
| The queue is consumed by a command line process called ``worker``.
|

> | The worker has to be always launched as it poll the queue waiting for new messages to handle.

Reviewer: polls

> | The worker has to be always launched as it poll the queue waiting for new messages to handle.

Reviewer 2: ```suggestion
| The worker has to always be launched as it polls the queue waiting for new messages to handle.
```
